workers/unity/Packages/com.improbable.gdk.core/Commands/WorldCommands.cs

```diff
@@ -86,6 +89,23 @@ public struct Request
                 public long RequestId;
             }
 
+            /// <summary>
+            ///     Method to create a CreateEntity command request payload.
+            /// </summary>
+            /// <param name="entity">
+            ///     The Entity object that defines the SpatialOS components on the to-be-created entity.
+            /// </param>
+            /// <param name="entityId">
+            ///     (Optional) The EntityId that the to-be-created entity should take.
+            ///     This should only be provided if received as the result of a ReserveEntityIds command.
+            /// </param>
+            /// <param name="timeoutMillis">
+            ///     (Optional) The command timeout in milliseconds. Will default to the WorkerSDK default.
```

Comment:
it will help to link to the definition of "workersdk default" in this param's doc

Reply:
and similar for other "timeoutMillis" too

Reply:
Actually this gets more confusing - there is a worker flag for adjusting the default command timeout (or 5s otherwise)